.github/workflows/verify.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@2226d7cb06a077cd73e56eedd38eecad18e5d837 # v3.1.0
         with:
-          version: v1.60
+          version: v1.64
           args: --timeout=5m
```

Comment:
this really ought to be fixed in separate prs by the dependabot and rebased away, as this is required but unrelated to mix in.